static/docs/commands-reference/update.md

```diff
@@ -13,22 +13,20 @@ positional arguments:
 
 ## Description
 
-After creating import stages ([DVC-files](/doc/user-guide/dvc-file-format)) with
-`dvc import`, the external data source can change. `dvc update` can be used to
-bring the imported <abbr>data artifact</abbr> up to date. This is the only
-command that does so with locked stages, and import stages are locked by
-default.
+After creating <abbr>import stages</abbr>
+([DVC-files](/doc/user-guide/dvc-file-format)) with `dvc import` or
+`dvc import-url`, the external data source can change. Use `dvc update` to bring
+these imported file, directory, or <abbr>data artifact</abbr> up to date.
+
+Note that import stages are locked by default. `dvc update` is the only single
+command that updates locked stages.
```

Comment:
@shcheklein re

> For each DVC-file:
>
> 1. It pings the URL (import-url) or checks remote Git repo (in case of import) that are specified in DVC-file created by dvc import/dvc import-url to see if the new version available (by comparing artifact checksum or rev-lock).
> 2. If the new version available it downloads and replaces the existing one with a new one.
> 3. It saves data to cache. Depending on cache strategy either by copying or by utilizing links.
> 4. It updates the DVC-file with the new checksum and rev-lock in case of dvc import.
>
> ... remove it, just because we are going to change the implementation logic - it won't depend on lock: true anymore soon.

OK note that I already removed the references to `locked: true` in 634ed43. But it seems I should just remove these notes completely. I'm just curious: how will import stages be locked moving forward if no `locked: true` will be used? For `dvc status` and `dvc repro` I mean.

Reply:
Moved discussion to https://github.com/iterative/dvc.org/pull/509#pullrequestreview-270476714

Reply:
@jorgeorpinel they will be locked by "definition". DVC is aware about their type, that they are created with import/import-url and repro won't affect them no matter what you do. At least for now. The only way to update them will be `dvc update`.

Reply:
I see. I'm still curious how DVC knows they're import stages. I guess given the fields used in the `deps` section of the DVC-file. Alright, Updated in f9e1d76! It was a relatively big change because one of the examples in `dvc import-url` had to be reviewed and partially re-written.